.github/workflows/lighthouse.yml

```
@@ -0,0 +1,213 @@
+                reportCount = 1;
+                console.log('개별 리포트에서 추출된 점수들:', scores);
+              }
+              
+              const getEmoji = (score) => {
+                if (score >= 90) return '🟢';
+                if (score >= 50) return '🟡';
+                return '🔴';
+              };
+              
+              const comment = `## 🚨 \`${appName}\` 라이트하우스 CI 결과 (Lighthouse CI Results)
+              
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)